salt/transport/zeromq.py

```diff
@@ -500,6 +617,10 @@ def __init__(self, opts, addr, linger=0, io_loop=None):
                            http://api.zeromq.org/2-1:zmq-setsockopt [ZMQ_LINGER]
         :param IOLoop io_loop: A Tornado IOLoop event scheduler [tornado.ioloop.IOLoop]
         """
+        salt.utils.versions.warn_until(
```

Comment:
dunder deprecate

Reply:
Only works at the module level with things being imported by the loader.